Compiling fails with error

When compiling a new SwiftPM project with your library it fails with the following errors

```
<ProjectPath>/.build/checkouts/SwiftDiscord--7968038837483286921/Sources/SwiftDiscord/Voice/DiscordVoiceEngine.swift:235:96: error: cannot convert value of type '[UInt8]' to expected argument type 'UInt8'
        let success = crypto_secretbox_easy(encrypted, &buf, UInt64(buf.count), &nonce, &secret!)
                                                                                         ~~~~~~^
<ProjectPath>/.build/checkouts/SwiftDiscord--7968038837483286921/Sources/SwiftDiscord/Voice/DiscordVoiceEngine.swift:255:114: error: cannot convert value of type '[UInt8]' to expected argument type 'UInt8'
        let success = crypto_secretbox_open_easy(unencrypted, voiceData, UInt64(data.count - 12), &nonce, &secret!)
                                                                                                           ~~~~~~^
error: terminated(1): /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swift-build-tool -f '<ProjectPath>/.build/debug.yaml' main output:
    


```

Here is my full output when I run the compile command: [https://pastebin.com/vUrpzfQS](https://pastebin.com/vUrpzfQS)

My main.swift only contains this:
```
import SwiftDiscord

print("Hello, world!")
```

First I used version 6.0.0 of your library, like suggested in the README.md, but now I tried 9.0.0, which doesn't work either. I am using Swift 4.

--- a/Sources/SwiftDiscord/Voice/DiscordVoiceEngine.swift
+++ b/Sources/SwiftDiscord/Voice/DiscordVoiceEngine.swift
@@ -232,7 +232,7 @@ public final class DiscordVoiceEngine : DiscordVoiceEngineSpec {
 
         defer { encrypted.deallocate() }
 
-        let success = crypto_secretbox_easy(encrypted, &buf, UInt64(buf.count), &nonce, secret!)
+        let success = crypto_secretbox_easy(encrypted, buf, UInt64(buf.count), nonce, secret)
 
         guard success != -1 else { throw EngineError.encryptionError }
 
@@ -252,7 +252,7 @@ public final class DiscordVoiceEngine : DiscordVoiceEngineSpec {
 
         defer { unencrypted.deallocate() }
 
-        let success = crypto_secretbox_open_easy(unencrypted, voiceData, UInt64(data.count - 12), &nonce, secret!)
+        let success = crypto_secretbox_open_easy(unencrypted, voiceData, UInt64(data.count - 12), nonce, secret)
 
         guard success != -1 else { throw EngineError.decryptionError }
 

--- a/Sources/SwiftDiscord/Voice/DiscordVoiceEngine.swift
+++ b/Sources/SwiftDiscord/Voice/DiscordVoiceEngine.swift
@@ -232,7 +232,7 @@ public final class DiscordVoiceEngine : DiscordVoiceEngineSpec {
 
         defer { encrypted.deallocate() }
 
-        let success = crypto_secretbox_easy(encrypted, &buf, UInt64(buf.count), &nonce, &secret!)
+        let success = crypto_secretbox_easy(encrypted, &buf, UInt64(buf.count), &nonce, secret!)
 
         guard success != -1 else { throw EngineError.encryptionError }
 
@@ -252,7 +252,7 @@ public final class DiscordVoiceEngine : DiscordVoiceEngineSpec {
 
         defer { unencrypted.deallocate() }
 
-        let success = crypto_secretbox_open_easy(unencrypted, voiceData, UInt64(data.count - 12), &nonce, &secret!)
+        let success = crypto_secretbox_open_easy(unencrypted, voiceData, UInt64(data.count - 12), &nonce, secret!)
 
         guard success != -1 else { throw EngineError.decryptionError }
 

